perfkitbenchmarker/providers/aws/aws_dynamodb.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2018 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module containing class for AWS' dynamodb tables.
+
+Tables can be created and deleted.
+"""
+
+import json
+import logging
+
+from perfkitbenchmarker import resource
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.providers.aws import util
+
+FLAGS = flags.FLAGS
+flags.DEFINE_enum('aws_dynamodb_attributetype',
+                  'S', ['S', 'N', 'B'],
+                  'The type of attribute, default to S, which is string.')
```

Comment:
add comment explain other types? N is number? B is binary?

Reply:
added suggested info